dnsone.go

```diff
@@ -27,6 +27,9 @@ func (s *ChallSrv) DeleteDNSOneChallenge(host string) {
 func (s *ChallSrv) GetDNSOneChallenge(host string) []string {
```

Comment:
Could you update this function's godoc comment to mention that cnames are considered/chased when looking up a TXT for the given `host`?

Reply:
Along similar lines I think it would be nice to add a mention of the new cname behaviour to the godoc for `GetDNSARecord`, `GetDNSAAAARecord`, and `GetDNSCAARecord`